trytravis.py

```diff
@@ -335,22 +408,26 @@ def _version_string():
 
 def _travis_headers():
     """ Returns the headers that the Travis API expects from clients. """
-    return {'User-Agent': ('trytravis/%s (https://github.com/'
-                           'SethMichaelLarson/trytravis)') % __version__,
-            'Accept': 'application/vnd.travis-ci.2+json'}
+    token = _load_travis_token()
+    return {'Travis-API-Version': '%s' % __version__,
+            'User-Agent': 'API Explorer',
```

Comment:
We shouldn't lie about our `User-Agent`. Please revert the user-agent change and hard-code the Travis-API-Version change.

Reply:
ok, done.